Projections/hydro_MacProjector.cpp

```diff
@@ -210,11 +234,12 @@ MacProjector::setLevelBC (int amrlev, const MultiFab* levelbcdata, const MultiFa
 {
     AMREX_ALWAYS_ASSERT_WITH_MESSAGE(!m_needs_domain_bcs,
                                      "setDomainBC must be called before setLevelBC");
+    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(
+        !robin_a || (robin_a && !m_abeclap),
+        "MacProjector::setLevelBC(): Projection only tested for case of EB separated mixed BC.");
```

Comment:
I don't think we can add this assertion. What if someone has already been using Robin BC for non-EB case?

Reply:
Fair enough (although I don't think the way the MAC Projector was before pr 122 would allow for robin bcs). I've just not tested the Mac Projector with anything other than EB separated cases. But if you say it's fine, then it's fine.

Reply:
In principle, all one needed to do before pr 122 was `getLinOp`.